spec/acceptance/iis_feature.rb

```diff
@@ -0,0 +1,50 @@
+require 'spec_helper_acceptance'
+
+describe 'iis_feature' do
+  context 'when managing features' do
+    context 'with default parameters' do
+      before(:all) do
+        @manifest  = <<-HERE
```

Comment:
I'm not sure this test is useful as pre-suit already installs the WebServer feature.  Perhaps we need to pick a non-default installed feature to test on.

Also as part of before(:all) may need to ensure that it does not exist before we test that we can install it.  Otherwise we're just testing two idempotent puppet runs, not an install puppet run and then and idempotent run.